packages/apputils-extension/src/index.ts

```diff
@@ -731,7 +739,7 @@ namespace Private {
   /**
    * The splash element.
    */
-  const splash = createSplash();
+  export let splash = createSplash();
```

Comment:
You don't use this anywhere, so it probably doesn't need to be exported in this PR, right? Or are you trying to address @Carreau's request here too of exporting some of these things?

Reply:
I fixed this too